LICENSE.txt

```diff
@@ -1,24 +1,9 @@
 BluePyOpt - Bluebrain Python Optimisation Library
 
-BluePyOpt is licensed under the LGPL, unless noted otherwise, e.g., for external 
-dependencies. See file LGPL.txt for the full license. 
-Examples and test are BSD-licensed.
```

Comment:
What happened with special case of examples/tests?

Reply:
What do you mean?

Reply:
The examples and tests have a different license (BSD) than the source code (LGPL)
The line stating that is removed with your changes.

Reply:
Then please disregard my changes. I was not aware a repo can have multiple licenses. Not sure how to handle those.

Reply:
I leave this PR open so you can make more changes? Or do I close it for now?

Reply:
Or maybe you can leave it open.

Reply:
Ok